File: tops-work/SE/Assignment/Module-2(Fundamentals-of-Programming)/String/que2.c
/*
Que2. Write a program in C to separate individual characters from a string.
*/

#include <stdio.h>
#include <string.h>

int main()
{
    char str[20];
    char temp;
    char reverse_str[20];
    printf("Enter any string\n");
    scanf("%s",&str);
    printf("The original string is\n%s\n",str);
    int len = strlen(str);
    for(int i = 0; i < len; i++)
    {
        printf("Character at index %d is: %c\n",i,str[i]);
    }
    return 0;
}

File: tops-work/PYTHON/Assignment/Fundamentals-of-Python/que2.py
"""
Que2. Write a Python program to get the Factorial number of given number.
for eg. factorial of 5
        5*4*3*2*1 = 120
"""

fact = 1
number = int(input("Enter any number: "))
for number in range(number,1,-1):
    fact *= number
    number = number - 1
print(fact)